NEWS.md

```diff
@@ -1,5 +1,9 @@
 # volcalc (development version)
 
+# volcalc 1.0.2
+
+* Minor change in calculation in `calc_vol()`---remove amines functional group to avoid double counting with primary amines
```

Comment:
```suggestion
* Minor change in calculation in `calc_vol()`---remove amines functional group to avoid double counting with primary amines (#49)
```

When there's useful context or conversation about a change, I think it's nice to reference the issue number in NEWS